indexer_util/indexer_util/indexer_util.py

```diff
@@ -41,7 +41,13 @@ def _convert_to_index_name(s):
     """Converts a string to an Elasticsearch index name."""
     # For Elasticsearch index name restrictions, see
     # https://github.com/DataBiosphere/data-explorer-indexers/issues/5#issue-308168951
-    prohibited_chars = [' ', '"', '*', '\\', '<', '|', ',', '>', '/', '?']
+    # Elasticsearch allows single quote in index names. However, they cause other
+    # problems. For example,
+    # "curl -XDELETE http://localhost:9200/nurse's_health_study" doesn't work.
```

Comment:
Where are the single quotes in this example?

Reply:
In nurse's

Reply:
Normally I would say to try escaping the punctuation, but if this is only going to be the backend index name and not the display name I think it's fine